ui/src/message_popup/message_popup_page.jsx

```diff
@@ -79,23 +87,54 @@ export default React.createClass({
       name: '',
       hasStarted: false,
       questionsAnswered: 0,
+      sessionLength: 20,
+      toastRevision: false,
+      responseTimes: []
     };
   },
 
   onStartPressed() {
-    const {competencyGroupValue, isSolutionMode} = this.state;
-    const questions = (isSolutionMode || competencyGroupValue === ALL_COMPETENCY_GROUPS)
-      ? _.shuffle(withStudents(allQuestions))
-      : questionsForCompetencies(competencyGroupValue);
-
+    const {competencyGroupValue} = this.state;
```

Comment:
It looks like this line isn't needed anymore, since this was moved to `getQuestions`?